shared/src/androidMain/kotlin/com/intive/picover/shared/di/SharedModule.kt

```diff
@@ -38,4 +35,9 @@ object SharedModule {
 	@Provides
 	fun provideWorkManager(@ApplicationContext context: Context) =
 		WorkManager.getInstance(context)
+
+	@Provides
+	@Singleton
+	fun provideToastPublisher(@ApplicationContext context: Context) =
+		ToastPublisher(context)
```

Comment:
Commit message is a bit misleading, actually koin is not involved in providing `ToastPublisher` 😄

Reply:
You're right, my bad 😞